### PR TITLE
Enhancement/icokpy/azure auth

### PR DESCRIPTION
**Pull request by liliankasem**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -43,6 +43,8 @@ oauthlib==2.0.2
 # OAuthlib version that logs errors - use until Flask-OAuthlib > 0.9.3 is published
 https://github.com/lepture/flask-oauthlib/archive/d62bf665fa2a4ab71327cd665334fdec25ade116.zip
 
+pyjwt==1.6.1
+
```
Comment (navjotk) [inline requirements.txt]: Does this replace another package that was previously used?

File: server/controllers/auth.py [hunk 2]
```diff
@@ -22,34 +25,34 @@
...
 auth = Blueprint('auth', __name__)
-
 auth.config = {}
 
+app = None
+provider_auth = None
+provider_name = None
+
+oauth = OAuth()
+
```
Comment (navjotk) [inline server/controllers/auth.py]: Would it be possible to refactor this to not require globals?

File: requirements.txt (hunk 1 above)
Comment (liliankasem) [reply requirements.txt]: nope, it does not :)

File: server/controllers/auth.py (hunk 2 above)
Comment (cicorias) [reply server/controllers/auth.py]: without moving their oauth into a getter/setter as well, no.

**Pull request merged**